### ISSUE TITLE
Backup: Sometimes Error: Failed to process metadata: System.IO.FileNotFoundException

### ISSUE THREAD

**Issue by martin54** [duplicati/duplicati#1911]
I have:
- [x] searched open and closed issues for duplicates

----------------------------------------

### Version info

**Duplicati Version:** 2.0.1.17_canary_2016-05-14
**Operating System:** Win.7 64 bit
**Backend:** Google Drive

### Bug description
(Multiple) Warning/Error message during backup on the "Log Data" web page  like the next one (http://127.0.0.1:8200/ngax/index.html#/log/1):
`Failed to process metadata for "<file>", storing empty metadata`
`System.IO.FileNotFoundException: <file>`
`{"ClassName":"System.IO.FileNotFoundException","Message":"<file>","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   bei System.Security.AccessControl.NativeObjectSecurity.CreateInternal(ResourceType resourceType, Boolean isContainer, String name, SafeHandle handle, AccessControlSections includeSections, Boolean createByName, ExceptionFromErrorCode exceptionFromErrorCode, Object exceptionContext)\r\n   bei System.Security.AccessControl.FileSystemSecurity..ctor(Boolean isContainer, String name, AccessControlSections includeSections, Boolean isDirectory)\r\n   bei System.Security.AccessControl.FileSecurity..ctor(String fileName, AccessControlSections includeSections)\r\n   bei Duplicati.Library.Snapshots.SystemIOWindows.GetAccessControlFile(String path)\r\n   bei Duplicati.Library.Snapshots.SystemIOWindows.GetMetadata(String path)\r\n   bei Duplicati.Library.Snapshots.WindowsSnapshot.GetMetadata(String file)\r\n   bei Duplicati.Library.Main.Operation.BackupHandler.GenerateMetadata(ISnapshotService snapshot, String path, FileAttributes attributes)","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":"8\nCreateInternal\nmscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089\nSystem.Security.AccessControl.NativeObjectSecurity\nSystem.Security.AccessControl.CommonSecurityDescriptor CreateInternal(System.Security.AccessControl.ResourceType, Boolean, System.String, System.Runtime.InteropServices.SafeHandle, System.Security.AccessControl.AccessControlSections, Boolean, ExceptionFromErrorCode, System.Object)","HResult":-2147024894,"Source":"mscorlib","WatsonBuckets":null,"FileNotFound_FileName":null,"FileNotFound_FusionLog":null}`

### Steps to reproduce
- only happens occasionally during backup, in my case typically related to Chrome files when Chrome is running in parallel and probably deleting some files out of ...\AppData\Local\Google\Chrome\User Data\Default\Local Extension Settings\\... .
- FYI: I'm using snapshots for backup.

**Assumed error scenario:** Another application is deleting files when Duplicati is trying to perform a backup in parallel, which causes temporal inconsistencies between the different processing steps of Duplicati.

**Actual result:** On local disc the reported file is not available any more. Within the backup, the file is present (and restore was successful).

**Expected result IF USING SNAPSHOTS:**
- If the backup is based on a snapshot, I don't expect such inconsistencies at all, as all operations should be performed on the same constant snapshot.
- My assumption: the operation "Read file metadata" is accessing the live file system instead of the snapshot, and in the live file system the file was already deleted.

**Expected result if NOT using snapshots:**
- These temporal inconsistencies cannot be avoided, and it's a normal behavior from my p.o.v. . Therefore this situation should not generate a warning/error message, instead only some kind of information message.

**Comment by agrajaghh** [comment]
duplicate of #1693 (which I closed now, since your description is better :blush:)

**Comment by UbuntuRunner** [comment]
I have the same problem with backing up a windows shared folder from a Win10-client (mounted on the duplicati machine on: /mnt/BackupShare/), so it might not have been solved (I also read the #1693).

Although I filtered out the folder "/mnt/BackupShare/System Volume Information/" in the backup settings, I get the following warning at the end of each duplicate backup: 

"Warnings: [
    Failed to process metadata for "/mnt/BackupShare/System Volume Information/", storing empty metadata => Unable to access the file "/mnt/BackupShare/System Volume Information/" with method llistxattr, error: EACCES (13)"
